package.json

```diff
@@ -18,6 +18,7 @@
     "test": "mocha test/**/*.test.ts"
   },
   "dependencies": {
+    "@hapi/joi": "^17.1.0",
```

Comment:
Should this be a developer dependency?

Reply:
No, validation is needed at runtime. Anything that application needs to do the job should be defined as regular dependency.